vembrane/modules/table.py

```diff
@@ -86,10 +86,8 @@ def tableize_vcf(
             yield env.table()
 
 
-def get_header(args) -> Optional[List[str]]:
-    if args.header == "none":
-        return
-    elif args.header == "auto":
```

Comment:
What's the reasoning behind going from returning `None` to returning `"none"`?

Reply:
It's actually a switch to not handle "none" in `get_header()`, but outside of it. Returning `None` here tripped up the csv.writer in this statement:
```
writer.writerow(get_header(args))
```
So I think it makes more sense to only trigger the header line writing statement when `args.header != "none"`.

Reply:
Also, the previous test to do the header write  (`if args.header:`) simply returns `True` for the input string `"none"`. So this had to be changed anyways. And then the `get_header()` function doesn't have to handle the `"none"` case any more.

Reply:
Ah, I see, very good.